apps/dashboard/redirects.js

```diff
@@ -1,6 +1,33 @@
 /** @type {import('next').NextConfig['redirects']} */
 function redirects() {
   return [
+    {
+      source: "/:network/connect-sdk",
+      destination: "https://portal.thirdweb.com/connect",
+      permanent: true,
```

Comment:
we never use permanent redirects because if we change our mind users will be stuck with these

Reply:
These changes are for when a user clicks on the icon of a product on the chainlist page. Such as Connect, Engine, Pay, etc. Currently they will 404. 

<img width="1270" alt="Screenshot 2024-09-16 at 10 08 08 AM" src="https://github.com/user-attachments/assets/c47dea39-46f9-41f0-8d1a-e39f7573270e">



For permanent: 
Won't permanent tell crawlers and search engines to use the new destination link? If we ever have to update these again with permanent = true, these would be overridden as well. 

Happy to change if not the case.

Reply:
if that's what they are for lets instead simply change the links on those, redirects are for when we can't solve it otherwise